src/main/java/io/github/cottonmc/cotton/gui/widget/WScrollBar.java

```diff
@@ -22,6 +32,28 @@ public class WScrollBar extends WWidget {
 	protected int anchor = -1;
 	protected int anchorValue = -1;
 	protected boolean sliding = false;
+	private static final BackgroundPainter background = BackgroundPainter.createLightDarkVariants(
+			createNinePatch(new Identifier(LibGuiCommon.MOD_ID, "textures/widget/scroll_bar/scroll_bar_background.png")),
+			createNinePatch(new Identifier(LibGuiCommon.MOD_ID, "textures/widget/scroll_bar/dark/scroll_bar_background.png"))
+	);
+
+	private static final BackgroundPainter scrollBar = BackgroundPainter.createLightDarkVariants(
+			createNinePatch(new Identifier(LibGuiCommon.MOD_ID, "textures/widget/scroll_bar/scroll_bar.png")),
+			createNinePatch(new Identifier(LibGuiCommon.MOD_ID, "textures/widget/scroll_bar/dark/scroll_bar.png"))
+	);
+
+	private static final BackgroundPainter scrollBarPressed = BackgroundPainter.createLightDarkVariants(
+			createNinePatch(new Identifier(LibGuiCommon.MOD_ID, "textures/widget/scroll_bar/scroll_bar_pressed.png")),
+			createNinePatch(new Identifier(LibGuiCommon.MOD_ID, "textures/widget/scroll_bar/dark/scroll_bar_pressed.png"))
+	);
+
+	private static final BackgroundPainter scrollBarHovered = BackgroundPainter.createLightDarkVariants(
+			createNinePatch(new Identifier(LibGuiCommon.MOD_ID, "textures/widget/scroll_bar/scroll_bar_hovered.png")),
+			createNinePatch(new Identifier(LibGuiCommon.MOD_ID, "textures/widget/scroll_bar/dark/scroll_bar_hovered.png"))
+	);
+
+	private static final NinePatchBackgroundPainter focus = createNinePatch(new Identifier(LibGuiCommon.MOD_ID, "textures/gui/scroll_bar/scroll_bar_focus.png"));
```

Comment:
1. Static final fields should be in `CONSTANT_CASE`
2. This code will fail to load on a dedi server. You should store these in a `private static final` inner class annotated with `@Environment(CLIENT)` like in `WTabPanel`:
https://github.com/CottonMC/LibGui/blob/e7a36835ff95ac045bbeac156a105c16b4d23df2/src/main/java/io/github/cottonmc/cotton/gui/widget/WTabPanel.java#L442-L456